pubspec.lock

```diff
@@ -392,4 +410,4 @@ packages:
     source: hosted
     version: "2.1.13"
 sdks:
-  dart: ">=1.24.0 <2.0.0-dev.infinity"
+  dart: "2.0.0-dev.3.0"
```

Comment:
When I run `pub upgrade` locally, for this line I get the following, rather than just `dart: "2.0.0-dev.3.0"`:

> dart: ">=2.0.0-dev.3.0 <=2.0.0-dev.8.0"

Reply:
Good catch, thanks.